Modules/Synchronization/src/Synchronization.Application/Datawallets/DTOs/PushDatawalletModificationItem.cs

```diff
@@ -1,11 +1,12 @@
 using Backbone.BuildingBlocks.Application.FluentValidation;
+using Backbone.Modules.Synchronization.Domain.Entities;
 using FluentValidation;
 
 namespace Backbone.Modules.Synchronization.Application.Datawallets.DTOs;
 
 public class PushDatawalletModificationItem
 {
-    public required DatawalletModificationDTO.DatawalletModificationType Type { get; set; }
+    public required DatawalletModificationType Type { get; set; }
```

Comment:
Using a separate enum here was intended. The reason is that we don't want the public API to change when we change an enum of our Domain project.